metals/src/main/scala/scala/meta/internal/metals/watcher/ProjectFileWatcher.scala

```diff
@@ -28,25 +28,25 @@ import com.swoval.files.PathWatchers
 import com.swoval.files.PathWatchers.Event.Kind
 
 /**
- * Watch selected files and execute  a callback on file events.
+ * Watch selected files and execute a callback on file events.
  *
- * This class recursively watches selected directories and selected files.
- * File events can be further filtered by the `watchFiler` parameter, which can speed by watching for changes
- * by limiting the number of files that need to be hashed.
+ * This class recursively watches selected directories and selected files. File events can be further filtered by the
+ * `watchFiler` parameter, which can speed by watching for changes by limiting the number of files that need to be
+ * hashed.
  *
  * We don't use the LSP dynamic file watcher capability because
  *
- * 1. the glob syntax is not defined in the LSP spec making it difficult to deliver a
- *    consistent file watching experience with all editor clients on all operating systems.
- * 2. we may have a lot of file watching events and it's presumably less overhead to
- *    get the notifications directly from the OS instead of through the editor via LSP.
+ *   1. the glob syntax is not defined in the LSP spec making it difficult to deliver a consistent file watching
+ *      experience with all editor clients on all operating systems. 2. we may have a lot of file watching events and
+ *      it's presumably less overhead to get the notifications directly from the OS instead of through the editor via
+ *      LSP.
  */
 final class ProjectFileWatcher(
-    config: MetalsServerConfig,
-    workspaceDeferred: () => AbsolutePath,
-    buildTargets: BuildTargets,
-    watchFilter: Path => Boolean,
-    onFileWatchEvent: FileWatcherEvent => Unit,
+  config: MetalsServerConfig,
```

Comment:
These formatting changes make reviewing more difficult. It'd be nice if these could be in a separate commit in the future.

Reply:
I agree. I think my editor was auto-formatting with our main scalafmt. A nice-to-have for metals if/when we have time would be per-project config and/or automatic scalafmt.conf detection (which would probably work everywhere but main :sweat_smile:)

Reply:
Fyi, I squashed the fixup format commit back onto this one so it doesn't show my bad formatting on it, and I actually formatted my next one before I pushed.